gunicorn/arbiter.py

```diff
@@ -494,6 +496,11 @@ def manage_workers(self):
 
     def spawn_worker(self):
         self.worker_age += 1
+        worker_id = -1
+        for i in range(self.num_workers):
```

Comment:
Having so many workers on one machine that this would actually be a performance issue is not a use case I'm familiar with, but I accept that it's possible.

Would just using a priority queue for the worker IDs be better?

Another thing that just occurred to me. If you TTOU signal and cause the number of workers to drop is it necessary to kill the worker with the highest ID, or is it okay to then have some worker(s) around with IDs higher than the number of workers?